tests/Feature/Auth/RegisterTest.php

```diff
@@ -55,8 +39,8 @@ public function it_does_allow_auth_user_to_register_new_users()
         $this->getJson(route('api.auth.user'))
             ->assertOk()
             ->assertJsonFragment([
-                'email' => $this->utility->user->email,
-                'name' => $this->utility->user->name,
+                'email' => $this->user->email,
```

Comment:
It was a bit confusing to understand where `$this->user` was coming from. Maybe it would be clearer if the user was returned from `$this->loginAdmin();` and then you can save it to a variable to be used in the test?